# POOH 2023 Metagenomics Workshop
This repository holds useful information for the workshop. This page also includes additional scrips useful for working through workshop materials on your own.
<p align="center">
  <img src="https://github.com/POOH-at-UCDavis/metagenomics_workshop/blob/429815b55d079b3e3ed53f5c0e15530ea52d5695/images/metagenomics_icon.png" width="600" height="400">
</p>

## Code of Conduct
Please abide by The Carpentries Code of Conduct for this workshop. 
This workshop is a community building initiative for metagenomics researchers on campus, and we ask that you support our community by being respectful to other learners. In particular, note that this is not an intellectual contest, we're all here to learn (including our instructional team!), and everyone has something valuble to contribute to our research community.

For details, see the [Code of Conduct Detailed View.](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html#the-carpentries-code-of-conduct)

# Schedule
We will be working through the Carpenteries Metagenomics Workshop; all the materials can be found on their [main page](https://carpentries-lab.github.io/metagenomics-workshop/)



| Time     | Topic                                                                                                                                                          | Presenter |
|----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|
|**Day 1**|                                                                                                                                                                 |           |
| 10:00 AM | Welcome                                                                                                                                                        | Hannah    |
| 10:10 AM | [Project Organization and Management](https://carpentries-lab.github.io/metagenomics-organization/)                                                            | Cory      |
| 10:50 AM | Intro to cluster and ssh                                                                                                                                       | Cory      |
| 11:00 AM | [Introduction to the Command Line](https://carpentries-incubator.github.io/shell-metagenomics/) Part 1-2                                                       | Dino      |
| 12:00 PM | Lunch Break                                                                                                                                                    | -         |
| 1:00 PM  | [Introduction to the Command Line](https://carpentries-incubator.github.io/shell-metagenomics/) Part 3                                                         | Dino      |
| 3:00 PM  | [Introduction to the Command Line](https://carpentries-incubator.github.io/shell-metagenomics/) Part 4-6                                                       | Bry       |
| 4:00 PM  | [Introduction to R](https://carpentries-incubator.github.io/introduction-to-R-for-metagenomics/)                                                               | Hugo      |
|**Day 2** |                                                                                                                                                                |           |
| 10:00 AM | Intro to Conda, scp, and srun                                                                                                                                  | Hannah    |
| 10:15 AM | [Starting a Metagenomics Project](https://carpentries-lab.github.io/metagenomics-analysis/01-background-metadata/index.html) & [Assessing Read Quality](https://carpentries-lab.github.io/metagenomics-analysis/02-assessing-read-quality/index.html) | Bry       |
| 11:15 AM | [Trimming and Filtering](https://carpentries-lab.github.io/metagenomics-analysis/03-trimming-filtering/index.html)                                             | Dino      |
| 12:00 PM | Lunch Break                                                                                                                                                    | -         |
| 1:00 PM  | [Metagenome Assembly](https://carpentries-lab.github.io/metagenomics-analysis/04-assembly/index.html)                                                          | Dino      |
| 1:30 PM  | [Metagenome Binning](https://carpentries-lab.github.io/metagenomics-analysis/05-binning/index.html) & [Taxonomic Assignment](https://carpentries-lab.github.io/metagenomics-analysis/06-taxonomic/index.html) | Cory      |
| 3:00 PM  | [Exploring Taxonomy with R](https://carpentries-lab.github.io/metagenomics-analysis/07-phyloseq/index.html), [Diversity Tackled With R](https://carpentries-lab.github.io/metagenomics-analysis/08-Diversity-tackled-with-R/index.html), & [Taxonomic Analysis with R](https://carpentries-lab.github.io/metagenomics-analysis/09-abundance-analyses/index.html) | Hannah    |
| 4:50 PM  | [Additional Resources](https://carpentries-lab.github.io/metagenomics-analysis/10-OtherResources/index.html)                                                   | Bry       |


## Where to go
The location is the [Shieldâ€™s library Datalab classroom (room 360)](https://datalab.ucdavis.edu/directions/). We recommend [parking in lot 5](https://goo.gl/maps/16CK2gMQizBsM6an8)


## Instructional Team
- Dino Sbardellati
  - PhD Candidate in Vannette Lab
  - Metagenomics, Microbial Ecology, Viral Ecology, Bees, R, Unix
- Cory Schlesener
  - PhD Candidate in Weimer Lab
  - Metagenomics, Microbial Ecology, Cows, Python, Unix
- Hannah Houts
  - PhD Candidate in DIB Lab
  - Metagenomics, Viral Ecology, Microbial Ecology, Human Gut, R, Unix, VS Code
- Bryshal Moore
  - PhD Candidate in DIB Lab
  - Metagenomics, 16s, Microbial Ecology, Human Gut, R, Unix
- Gina Vazquez
  - PhD Student in DIB Lab
  - Metagenomics, Microbial Ecology, Biosynthetic gene clusters, R, Unix, 
- Hugo F. Monteiro
  - Postdoc at the School of Veterinary Medicine
  - Gut Fermentation, Microbial Ecology, Amplicon, Metagenomics, Metatranscriptomics, Metabolomics, R, Unix, Python, SAS, AI & ML
- Titus Brown
  - Professor in Population Health & Reproduction; Datalab Director for Community Engagement; PI of DIB Lab
  - 'omics methods development, microbial ecology, strain diversity, python, unix, 


## Onboarding - Before you arive
please complete this content before we meet on Tuedsay
###  Install BOTH R and RStudio
Download and install R from CRAN here: https://cran.r-project.org/
Download and install the IDE RStudio: https://posit.co/download/rstudio-desktop/
###  Install a bash terminal with mobaxterm (Windows users only)
[Tutorial](https://mobaxterm.mobatek.net/)
### Please review this video about a cluster
(Lecture to ~25 min, remaining ~35 min demo is optional)
https://ucdavis.box.com/s/l2e7he830wirskct2rje86ftblao8fbd



